### DISCUSSION TITLE
How to authenticate  to api with Azure B2C?

### DISCUSSION THREAD

**Discussion by biapar** [directus/directus#8074] [Archive]
How to authenticate  to api with Azure B2C?
Thanks.

**Comment by rijkvanzanten** [comment]
Seems like Azure Active Directory B2C supports oAuth2, so you should be able to configure it as an oauth provider. 

https://docs.directus.io/reference/environment-variables/#oauth
https://docs.microsoft.com/en-us/azure/active-directory-b2c/protocols-overview
